libs/partners/prompty/langchain_prompty/langchain.py

```diff
@@ -0,0 +1,19 @@
+from langchain_core.prompts import ChatPromptTemplate, MessagesPlaceholder
+from langchain_core.runnables import RunnableLambda, Runnable
+from typing import Dict
+from .utils import load, prepare
+
+def create_chat_prompt(path: str, input_name_agent_scratchpad = "agent_scratchpad") -> Runnable[Dict[str, ChatPromptTemplate], str]:
```

Comment:
my bad, typo here, it should be 
```
Runnable[Dict[str, any], ChatPromptTemplate]
```

I'm basically constructing a ChatPromptTemplate using input and prompty. Prompty will render and parse based on user inputs.

Reply:
What would a user pass in that's different than what they'd pass into a ChatPromptTemplate? Would it be possible to have it return a ChatPromptTemplate instead of a RunnableLambda that returns a ChatPromptTemplate?